### ISSUE TITLE
Missing Web-Settings 

### ISSUE THREAD

**Issue by C00Lzero** [cdgco/VestaWebInterface#89]
### OS / Environment
```
Distributor ID: Debian
Description:    Debian GNU/Linux 8.11 (jessie)
Release:        8.11
Codename:       jessie
Linux <myname> 3.16.0-7-amd64 #1 SMP Debian 3.16.59-1 (2018-10-03) x86_64 GNU/Linux
PHP 7.1.26-1+0~20190113101856.12+jessie~1.gbp7077bb (cli) (built: Jan 13 2019 15:14:34) ( NTS )
```

### Summary of the problem
Web-Settings not available, even though it's activated in the settings (and database)
Everything else seems to work.
![missing_web_settings](https://user-images.githubusercontent.com/1133827/58800949-81adf600-8609-11e9-96bd-131027dd496a.PNG)
![missing_web_settings_large](https://user-images.githubusercontent.com/1133827/58800950-82468c80-8609-11e9-9eab-71443ef30867.PNG)

### Steps to reproduce the behavior
just a fresh installation of the vwi with the panel version 0.9.8-24

### Source code / logs / error code
PHP-Messages: 
```
PHP message: PHP Notice:  Undefined variable: recordcount in /home/admin/web/<domain>/public_html/index.php on line 510
PHP message: PHP Notice:  Undefined variable: requestdns in /home/admin/web/<domain>/public_html/index.php on line 443" while reading upstream, client: x.x.x.x, server: panel.ookami.services, request: "GET /index.php HTTP/1.1", upstream: "fastcgi://<cgi>", host: "<domain>", referrer: "https://<domain>/list/backups.php"
2019/06/03 14:07:13 [error] 10416#10416: *901676 FastCGI sent in stderr: "PHP message: PHP Notice:  Undefined offset: 1 in /home/admin/web/<domain>/public_html/index.php on line 861" while reading upstream, client: x.x.x.x, server: <domain> request: "GET /index.php HTTP/1.1", upstream: "fastcgi://<cgi>", host: "<domain>", referrer: "https://<domain>/list/backups.php"
```
Though only the offset seems like a small problem...
 not sure, why it's missing :(

I hope someone can help me here. 
Thanks

**Comment by cdgco** [comment]
VWI checks in the background to see if either apache2, nginx or php-fpm is installed, if it does not detect those servers it disables the web system. There is likely an error in the detection. I am not at my computer at the moment but when I get a chance I will write a command for you to run to check if that is the error.

**Comment by C00Lzero** [comment]
ah! this is the issue. 
I've upgraded the installed php5 to php7
put there the processes are called php<version>-fpm; so the detection won't return a match. 

i've come up with 3 possibilities: 
1) change the checker to '-fpm' only 
2) change the checker to preg_match
3) change the checker to check first for trailing / (as for regex) in the String and then decide what to do. 

on my copy I got with nr. 3. since strpos is faster. though i'm not sure if the check for the first character could be faster
i've created a pull-request: #90 

also I close this question, since it's now working. 

Thank you.
